views/plugin/basemap/basemaplayerui.html

```diff
@@ -53,7 +53,7 @@
         <tr title="Sets the minimum zoom level at which the layer will display">
           <td class="text-nowrap">Min Zoom:</td>
           <td class="w-100">
-            <spinner min="2" max="maxZoom" step="1" value="minZoom" name="minZoom"></spinner>
+            <spinner min="::basemap.mapMinZoom" max="maxZoom" step="1" value="minZoom" name="minZoom"></spinner>
```

Comment:
In angular 1.3+ `::` creates a one time binding. the `$watch` is automatically deregistered after the first resolved value. Since map max/min doesn't change during run time there is no need to `$watch` for changes